test/e2e/xsite/xsite_test.go

```diff
@@ -231,6 +231,16 @@ func TestDefaultTLSLoadBalancerWithPort(t *testing.T) {
 	testCrossSiteView(t, true, ispnv1.CrossSiteSchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeLoadBalancer, 1443, 1, DefaultTLS, nil)
 }
 
+func TestCrossSiteViewOpenshiftRoute(t *testing.T) {
```

Comment:
Having internal version of the test for smoke testing on a single cluster would be nice

Reply:
that is useless. Cross-site in a single cluster uses the cluster IP directly and not the Routes (same for NodePort & LoadBalancer)

Reply:
Right, I incorrectly though we have "single cluster" version of the tests for NodePort and LoadBalancer too.